docs/src/docs/asciidoc/user/eviction-advisor.adoc

```diff
@@ -14,37 +14,39 @@ endif::notBuildingForSite[]
 
 NOTE: This is an advanced topic/feature that will not be of interest to most users.
 
-You can affect which elements are selected for eviction from the cache by providing a class that implements the
-`org.ehcache.config.EvictionAdvisor` interface.
+You can affect which elements are selected for eviction from the cache by providing a class that implements the `org.ehcache.config.EvictionAdvisor` interface.
 
-NOTE: Eviction advisors are not used for clustered storage tiers.
-For example, in a cache with a heap tier and clustered storage tier,
-the heap tier will use the eviction advisor but the clustered storage tier will evict independently, irrespective of the eviction advisor.
+[NOTE]
+====
+Eviction advisors are not used for clustered storage tiers.
+For example, in a cache with a heap tier and clustered storage tier, the heap tier will use the eviction advisor but the clustered storage tier will evict independently, irrespective of the eviction advisor.
 The description below applies to using an eviction advisor for the cache tiers other than a clustered storage tier.
+====
 
-`EvictionAdvisor` implementations are invoked when Ehcache attempts to evict entries from the cache
-(in order to make room for new entries) in order to determine whether the given entry should not be considered
-a good candidate for eviction.  If the eviction is advised against, Ehcache will try to honor the preference of
-preserving that entry in the cache, though there is no full guarantee of such.
+`EvictionAdvisor` implementations are invoked when Ehcache is attempting to evict entries from the cache
+(in order to make room for new entries) in order to determine whether the given entry should not be considered a good candidate for eviction.
```

Comment:
General comment: these parenthesized comments might be more appropriately made 1st class citizens: ...when Ehcache is attempting to evict entries from the cache to make room for new entries, in order to determine whether the given entry...